turbo.json

```diff
@@ -0,0 +1,48 @@
+{
+  "pipeline": {
+    "build": {
+      "dependsOn": ["intl-compile", "codegen"],
+      "outputs": ["./dist/**"]
+    },
+    "codegen": {
+      "dependsOn": ["^codegen"],
+      "outputs": ["./src/api/**"]
+    },
+    "dev": {
```

Comment:
Looks like something went wrong with Hydrogen when I tried `npm run dev`
![image](https://user-images.githubusercontent.com/8978655/208940954-6122c80e-f1ad-4d66-b985-283070a21e68.png)

Reply:
hmm, I'm going to have to try this with a fresh repo. Can't seem to reproduce atm.

Reply:
i also see this for `npm run dev`:
![Screen Shot 2022-12-21 at 12 10 03](https://user-images.githubusercontent.com/3046459/208964050-4700a10d-a530-417c-9745-9467d3924e94.png)

Reply:
Okay, so I still wasn't able to reproduce but I updated the entry to use path resolution which should hopefully prevent this 🤔 

[make changes to hydrogen resolution](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5173/commits/480770f413e94122c6b524153bd6ed00c758f710)

Reply:
That worked for me (once I recreated webserver and such in docker). How about you @petertgiles?